gcloud_dataproc/submit.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
copied back from previously deleted file, cleaned up to  have a helper `submit` function as well as the main